BadMedicine.Core/Datasets/DataGenerator.cs

```diff
@@ -367,18 +402,18 @@ public string GetRandomSentence(Random r)
                 case 1: return "I often see the time 11:11 or 12:34 on clocks.";
                 case 2: return "Malls are great places to shop; I can find everything I need under one roof.";
                 case 3: return "Christmas is coming.";
-                case 4: return "I will never be this young again. Ever. Oh damn� I just got older.";
+                case 4: return "I will never be this young again. Ever. Oh damn� I just got older.";
```

Comment:
Oh dear, whats this creepy character

Reply:
Replaced these with regular single quotes in https://github.com/HicServices/BadMedicine/pull/3/commits/8c3f623f0af988c22922fa2cc42c5d89d62cda8e